libs/cvr-fixture-generator/jest.config.js

```diff
@@ -14,4 +14,5 @@ module.exports = {
       lines: 95,
     },
   },
+  prettierPath: null,
```

Comment:
Just curious: What does this change do?

Reply:
Sorry, I made a comment but forgot to "submit" my review. https://github.com/votingworks/vxsuite/pull/4445#discussion_r1430714110